stoplight.js

```diff
@@ -0,0 +1,6 @@
+$('div').on('click', function(changeColor){
+  $( '#stoplight').css('div','.bulb');
+  event.preventDefault(changeColor);
```

Comment:
don't need `event.preventDefault()` here. There's no default action to prevent.

you are close with `$( '#stoplight').css('div','.bulb')` 

[answers are for this section are here](https://github.com/ga-wdi-exercises/checkpoint-02/blob/solution/stoplight.js)